data/json/items/magazine/nail.json

```diff
@@ -0,0 +1,20 @@
+[
+  {	
+    "id": "nailmag",	
+    "looks_like": "stanag30",	
+    "type": "MAGAZINE",	
+    "name": { "str": "coilgun magazine" },	
+    "description": "An improvised magazine for use with a coilgun.  Little more than a metal box, spring and some duct tape it is awkward to reload and not especially reliable.",	
+    "weight": "60 g",	
+    "volume": "250 ml",	
+    "price": 1920,	
+    "material": "steel",	
+    "symbol": "#",	
+    "color": "light_gray",	
+    "ammo_type": "nail",	
+    "capacity": 50,	
```

Comment:
I don't think this will work, there's no pocket data.

Reply:
Added.